src/comments.js

```diff
@@ -46,7 +46,14 @@ export default class Mastodon extends HTMLElement {
   async fetchComments(url) {
     const tootUrl = new URL(url);
     const { origin, pathname } = tootUrl;
-    const [_, id] = pathname.match(/\/(\d+)$/);
+
+    let id = pathname.match(/\/(\d+)$/);
```

Comment:
This is not rigth. The result of `match` is an array. You have to get the second item (but using `let`):

```js
let [_, id] = pathname.match(/\/(\d+)$/);
```